airflow/utils/pydantic.py

```diff
@@ -24,19 +24,15 @@
 
 from __future__ import annotations
 
+from importlib.metadata import PackageNotFoundError, version
```

Comment:
```suggestion
from importlib.metadata import version
```
`PackageNotFoundError` is subclass of `ImportError`

Reply:
nice. Yep. Better to handle more generic case (though I think it does not matter in this case - I doubt metadata.version will raise any other "Import Error" :)